### DISCUSSION TITLE
OS Storage

### DISCUSSION THREAD

**Discussion by hoangpq14736** [anasty17/mirror-leech-telegram-bot#963] [Q&A]
Hi, Im totally noob here. I was successfully deploy the bot and I think that the bot did download then upload the file to team drive but it did not delete the file somewhere in the OS storage. So I wanna ask that where was the file located (deployed with Heroku), can I delete the file I've download in the OS and what's the function of /clearlocals?

**Comment by anasty17** [comment]
How did u know that the file not deleted?

**Reply by hoangpq14736** [reply]
Sorry to interrupt you.  I just scroll over the log and saw the Cleaning Download line.
